templates/guides/import-script.md

```diff
@@ -0,0 +1,132 @@
+---
+page_title: "Use Terraform Import Script to migrate existing resources"
+---
+
+# Use Import Script to migrate existing resources
+The Astro Terraform Import Script is a tool designed to help you import existing Astro resources into your Terraform configuration.
+
+In this guide, we will migrate an existing Workspace, API token and Team into Terraform using the Terraform Import Script.
+
+### Import Script options
```

Comment:
```suggestion
## Import Script Options
```

Reply:
Following the Astronomer docs style guide, we wouldnt capitalize "options" here. I will bump up the header level though.